packages/react/src/components/Alert/Alert.tsx

```diff
@@ -0,0 +1,134 @@
+import * as _ from 'lodash'
+import * as PropTypes from 'prop-types'
+import * as React from 'react'
+
+import {
+  UIComponent,
+  UIComponentProps,
+  ContentComponentProps,
+  commonPropTypes,
+  customPropTypes,
+  childrenExist,
+  rtlTextContainer,
+} from '../../lib'
+import { RenderResultConfig } from 'src/lib/renderComponent'
+import { defaultBehavior } from '../../lib/accessibility'
+import { Accessibility } from '../../lib/accessibility/types'
+import { ComponentEventHandler, ReactProps, ShorthandValue } from '../../types'
+import { Box, Icon } from '../..'
```

Comment:
Please use direct imports to avoid circular dependencies:

```
import Box from '../Box/Box`
import Icon from '../Icon/Icon`
```

Reply:
hmm..ok, this is what I get with automatic imports, why is that?

Reply:
Automatic imports sometimes are weird, this case I think you got it because `Box` in `src/index.ts` is named import while in `src/Box/Box.ts` it's default.

Reply:
done, thx